src/components/ConversationWindow.js

```diff
@@ -703,8 +793,9 @@ ConversationWindow.defaultProps = {
   allowGuestUsers: false
 };
 
-const mapStateToProps = ({message, client}, ownProps) => {
-  return {...message, ...client, ownProps: ownProps}
+const mapStateToProps = ({message, conversation, client}, ownProps) => {
+  // return {...message, ...client, ownProps: ownProps}
+  return {...message, ...conversation, ...client, ownProps: ownProps}
```

Comment:
@pardeep-kumar-bs why we have passed `...conversation` here as it will affect this component's performance and will do unnecessary rendering.

Reply:
Done.